src/XMakeBuildEngine/Evaluation/ProjectRootElementCache.cs

```diff
@@ -236,12 +236,17 @@ internal ProjectRootElementCache(bool autoReloadFromDisk)
                             // the cache as we get test coverage of the rest of the cache code.
                             XmlDocument document = new XmlDocument();
                             document.PreserveWhitespace = projectRootElement.XmlDocument.PreserveWhitespace;
-							
-                            XmlReaderSettings xmlReaderSettings = new XmlReaderSettings();
-                            xmlReaderSettings.DtdProcessing = DtdProcessing.Ignore;
-                            using (XmlReader xr = XmlReader.Create(File.OpenRead(projectRootElement.FullPath), xmlReaderSettings))
+
+                            XmlReaderSettings dtdSettings = new XmlReaderSettings();
+                            dtdSettings.DtdProcessing = DtdProcessing.Ignore;
+#if FEATURE_XMLTEXTREADER
```

Comment:
That's strange. This `if else` doesn't come from me and it's actually not in the `xplat` nor the `master` branch??? :confused: